public/404.html

```diff
@@ -1,67 +0,0 @@
-<!DOCTYPE html>
```

Comment:
@danielpaul The reason I deleted these files was because they pop up instead of the dynamic error files when you go to this page 'http://localhost:3000/404'